pages/dkp/kommander/2.1/install/networked/index.md

```diff
@@ -96,6 +96,12 @@ helmrelease.helm.toolkit.fluxcd.io/traefik-forward-auth-mgmt condition met
 helmrelease.helm.toolkit.fluxcd.io/velero condition met
 ```
 
+If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress", you can trigger a reconciliation of the `HelmRelease` using the following command:
+
+```bash
+kubectl annotate --overwrite helmrelease/<HELMRELEASE_NAME> -n <WORKSPACE_NAMESPACE> reconcile.fluxcd.io/requestedAt="$(date +%s)"
```

Comment:
This does not work if the number of retries is exhausted. See https://github.com/fluxcd/helm-controller/issues/366

I believe the suspend/resume trick works in that case but haven't tried.

Reply:
ah, thanks @makkes, will update

Reply:
Updated in https://github.com/mesosphere/dcos-docs-site/pull/4310/commits/edd9bc416d22f30b43cfc5fbda38493433cef88b